pom.xml

```diff
@@ -71,7 +71,7 @@
 
     <version.io.opentracing>0.31.0</version.io.opentracing>
     <version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>0.1.0</version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>
-    <version.io.opentracing.contrib-opentracing-spring-web>0.3.2</version.io.opentracing.contrib-opentracing-spring-web>
+    <version.io.opentracing.contrib-opentracing-spring-web>0.3.3</version.io.opentracing.contrib-opentracing-spring-web>
```

Comment:
oh we could cut a minor version

Reply:
I agree that cutting a new minor version for `java-spring-cloud` would probably be good